Source/ending.py

```diff
@@ -0,0 +1,96 @@
+# Author: Siena Okuno
+# Project: Interactive Programming
+# Date: 10.30.2017
+# Description: Determines endings
+
+import pygame
+import sys
+
+def screenScroll(screen, color):
+    '''Turns screen a color by filling in with rectangles.'''
+    for i in range(700):
+        pygame.draw.rect(screen, color, (0,i,1000,i+1))
+        i+=1
```

Comment:
The `i+=1` line doesn't do anything, since the next time through the loop resets `i`